web/shapeworks/src/views/Main.vue

```diff
@@ -409,7 +411,7 @@ export default {
                         <v-tab-item value="data">
                             <data-list :dataset="dataset" autoSelectOne/>
                         </v-tab-item>
-                        <v-tab href="#info">Info</v-tab>
+                        <v-tab v-if="selectedProject && selectedProject.landmarks.length > 0" href="#info">Info</v-tab>
```

Comment:
Can you explain this addition?

Reply:
I'm pretty confident this is a relic from before when landmarks was incomplete. I'll remove it